bin/helpers/config.js

```diff
@@ -1,16 +1,25 @@
 var config = {};
 config.env = process.env.NODE_ENV || "production";
 
-// load config based on env
-require('custom-env').env(config.env)
+if(config.env === "production") {
+  config.uploadUrl = 'https://api-cloud.browserstack.com/automate-frameworks/cypress/upload';
```

Comment:
Can we move these to config.json and load that JSON on line 1?

Reply:
done